ci: remove use of pull_request_target (part 1) (#3747)
GoogleCloudPlatform/nodejs-docs-samples: CONTRIBUTING.md

        npm test

### Running the tests for a Pull Request

> When a Pull Request is opened, reopened, or has new commits pushed the tests will be run.

Author: ```suggestion
When a Pull Request is opened, reopened, or has new commits pushed the sample tests (unit, integration, end-to-end) will be run.
```

Author: @iennae does this address clarifying the scope of "tests"?

Author: Will move forward with this and we can keep iterating.

> If the tests for a sample change do not run, they can be triggered by adding the `actions:force-run` label.

Reviewer: This only triggers a subset of tests right?

Author: Yes, this is for the sample tests, as distinguished from all validations/checks that are run.

Added a comment with a suggestion to clarify.

> If tests need to be triggered a second time, manually remove and re-add this label.

Reviewer: If you need tests to re-run multiple times (merging changes from main, updating changes based on feedback), manually remove the 'actions:force-run' and then re-add this label. 

(not just a second time, but any number more than 1, is there an amount of time that needs to be waited in between manually removing and then readding for it to get caught?)

Author: I've converted your feedback to a suggested change below, but to address a couple questions:

1. I've not noticed any particular wait time, but I usually wait for the label updated to be recorded in the comment/timeline view by GitHub.

2. If you merge changes from main the PR will automatically retest. The only times to use this are queueing is stuck or we think the failure was a flake or external bug.

```suggestion
If tests need to be triggered multiple times, manually remove `actions:force-run` and then re-add this label.
```

> The label will not be automatically removed. Please remove it before merging the Pull Request.

Reviewer: nit: The automatic clean-up of labels is currently disabled. Please remove the `actions:force-run` before merging the Pull Request. 

subtle change, make it explicit that we are intentionally not automatically removing right now. I'm in favor of repeating the exact label to take action on as I'm not assuming folks are not bouncing to exactly the area that they are looking to follow instructions for.

Author: ```suggestion
The automatic clean-up of labels is currently disabled. Please remove the actions:force-run before merging the Pull Request.
```
